frontend/class-frontend.php

```diff
@@ -346,13 +346,13 @@ public function get_default_title( $sep, $seplocation, $title = '' ) {
 		if ( ! is_string( $title ) || ( is_string( $title ) && $title === '' ) ) {
 			$title = get_bloginfo( 'name' );
 			$title = $this->add_paging_to_title( $sep, $seplocation, $title );
-			$title = $this->add_to_title( $sep, $seplocation, $title, strip_tags( get_bloginfo( 'description' ) ) );
+			$title = $this->add_to_title( $sep, $seplocation, $title, wp_strip_all_tags( get_bloginfo( 'description' ) ) );
```

Comment:
I understand the logic to not strip new lines from the description, but as this is used to build up the `title`, it should strip any new lines.

Line 577 also applies `wp_strip_all_tags` but as this method is public, doing it internally as well make sense.